### PR TITLE
Added basic UI and scripting for Strategy Cards

### PR DESCRIPTION
**Pull request by TenjouUtena**

### CODE REVIEW

**Comment by darrellanderson** [comment]
I'm not sure we want an explicit "reset".  Players often click play then are negotiated into waiting ... leaving the play button always active is a lot easier to understand should someone want to play it again during the same round.

**Comment by TenjouUtena** [comment]
I think that's fair.   I was trying to balance something like that with the idea that we maybe want to trigger some automation on pushing play, and we don't want to trigger it several times.   I'm happy to:
* back out the code for tracking the activation
* Create some option to control it
* Some other thing based on guidance
Let me know,   Thanks!

**Comment by darrellanderson** [comment]
Sounds good.  You can edit src/global.js to add an event:

```
globalEvents.TI4 = {
    ...
    // Called when a player clicks "play" on a strategy card.
    // <(object: strategy card, player: Player) => void>
    onStrategyCardPlayed : new TriggerableMulticastDelegate(),
}
```

then in the card:

```
tp.globalEvents.TI4.onStrategyCardPlayed.trigger(refObject, player)
```

I'm tempted to also suggest you make your code in `src/global/` and have it add a button to _any_ strategy card (detecting using the object namespace) so homebrew strategy cards don't need a script (nor to the "real ones").  That might be something to consider later.

File: assets/Templates/stratcards/base/0645434449AEC8D41E779DAB9C865D76.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"Type": "Generic",
 	"GUID": "AE7D6A5C4EEDAAE2B2FEB5AFBA3AC6A1",
 	"Name": "Construction",
-	"Metadata": "",
+	"Metadata": "card.strategy/construction",
```
Comment (darrellanderson) [inline assets/Templates/stratcards/base/0645434449AEC8D41E779DAB9C865D76.json:5]: I need to document the metadata namespace, this is actually "tile.strategy:base/construction".  I won't repeat this for most of the others, except where there's a notable tweak.

File: assets/Templates/stratcards/codex1/39364D824F5CB1EFAAE8928976925102.json [hunk 2]
```diff
@@ -22,10 +22,10 @@
 		"G": 0,
 		"B": 0
 	},
-	"Flippable": false,
+	"Flippable": true,
 	"AutoStraighten": false,
 	"ShouldSnap": true,
-	"ScriptName": "",
+	"ScriptName": "gameobjects/strategy-card.js",
```
Comment (darrellanderson) [inline assets/Templates/stratcards/codex1/39364D824F5CB1EFAAE8928976925102.json:28]: shouldn't have a script anymore

the Metadata here is "tile.strategy:codex.ordinian/diplomacy", keeping the ":base" version should someone choose to use it.

File: assets/Templates/stratcards/pok/0DFFBB93442C459587AD4296177FAE6C.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
 		"G": 0,
 		"B": 0
 	},
-	"Flippable": false,
+	"Flippable": true,
 	"AutoStraighten": false,
 	"ShouldSnap": true,
-	"ScriptName": "",
+	"ScriptName": "gameobjects/strategy-card.js",
```
Comment (darrellanderson) [inline assets/Templates/stratcards/pok/0DFFBB93442C459587AD4296177FAE6C.json:28]: shouldn't have a script anymore

the Metadata here is "tile.strategy:pok/construction", keeping the ":base" version should someone choose to use it.

File: src/lib/helpers.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+function isCommandToken(obj) {
```
Comment (darrellanderson) [inline src/lib/helpers.js:1]: I've got a slightly different approach to this, but let's adopt this one for now and consider the other style separately.

**Review by darrellanderson — APPROVED** [review]
Looks good, appreciate the tweaks!

**Pull request merged**